### PR TITLE
Added some more social media, gaming and fashion companies

### PR DESCRIPTION
**Pull request by nodeg**
- Add Logitech Europe S.A.
- Add CD PROJEKT S.A.
- Add Fashion ID GmbH & Co. KG
- Add Peek & Cloppenburg KG
- Add Rocket.Chat Technologies Corp.
- Add Signal Messenger LLC.
- Add Threema GmbH

I was not able to find telephone and fax numbers for some companies. Feel free to add missing parts.

### CODE REVIEW

**Comment by nodeg** [comment]
Hm, something seems wrong. I am looking into that.

**Comment by nodeg** [comment]
Yesterday I missed that Caseking GmbH runs noblechairs.de so I added that part.

File: companies/cdprojekt.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "slug": "cdprojekt",
+    "relevant-countries": [
+        "all"
+    ],
+    "name": "CD PROJEKT S.A.",
+    "runs": [
+        "GOG"
+    ],
+    "address": "ul. Jagiellońska 74\nbuilding E\n03-301 Warschau\nPolen",
```
Comment (baltpeter) [inline companies/cdprojekt.json]: When the company is relevant to non-German countries (as is the case here), we usually have the address in English.

File: companies/logitech.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "all"
+    ],
+    "name": "Logitech Europe S.A.",
+    "runs": [
+        "Astrogaming",
+        "Jaybird",
+        "Saitek",
+        "Ultimateears"        
+    ],
+    "address": "EPFL - Quartier de l'Innovation\nDaniel Borel Innovation Center\nCH-1015 Lausanne\nSchweiz",
```
Comment (baltpeter) [inline companies/logitech.json]: Same here.

File: companies/threema.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "slug": "threema",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication",
+        "social media"
+    ],
+    "name": "Threema GmbH",
+    "address": "Churerstrasse 82\n8808 Pfäffikon SZ\nSchweiz",
```
Comment (baltpeter) [inline companies/threema.json]: And one final time.

**Comment by nodeg** [comment]
Thanks for the heads up. I will keep that in mind.

**Pull request merged**